c_tests/tests/compile_globals.c

```diff
@@ -1,5 +1,6 @@
 // Compiler:
```

Comment:
I suspect this file should be called something like `mutable_global`?

Reply:
Agreed.

Reply:
Fixed in a19c0c8.